src/Nuget.PackageIndex/LocalPackageIndexBuilder.cs

```diff
@@ -136,7 +136,13 @@ public Task<LocalPackageIndexBuilderResult> BuildAsync(bool shouldClean = false,
                     {
                         if (cancellationToken != null && cancellationToken.IsCancellationRequested)
                         {
-                            return new LocalPackageIndexBuilderResult { Success = false, TimeElapsed = stopWatch.Elapsed }; ;
+                            return new LocalPackageIndexBuilderResult { Success = false, TimeElapsed = stopWatch.Elapsed };
```

Comment:
Hm i used to have cases when i needed null to be returned, will need to see if it can be a struct , but not here - in separate check in.